packages/contracts-bedrock/contracts/test/LibPosition.t.sol

```diff
@@ -144,9 +144,265 @@ contract LibPosition_Test is Test {
         _indexAtDepth = boundIndexAtDepth(_depth, _indexAtDepth);
 
         Position position = LibPosition.wrap(_depth, _indexAtDepth);
-        Position defend = position.defend();
+        Position defend = position.move(false);
 
         assertEq(defend.depth(), _depth + 1);
         assertEq(defend.indexAtDepth(), ((_indexAtDepth / 2) * 2 + 1) * 2);
     }
+
+    /**
+     * @notice A static unit test for the correctness of all gindicies, (depth, index) combos,
+     *         and the trace index in a tree of max depth = 4.
+     */
+    function test_pos_correctness() public {
```

Comment:
Style guide asks for specific suffixes, should use either `succeeds` or `works` if you're just testing correctness.
see: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/STYLE_GUIDE.md#contract-naming-conventions
```suggestion
    function test_pos_succeeds() public {
```

Reply:
Good looks 👍